docs/source/concepts/time.rst

```diff
@@ -11,7 +11,52 @@ Thinking about Time in the Simulation
+desired step size is not necessarily known in advance. Therefore, more commonly used is the :class:`DateTimeClock <vivarium.framework.time.DateTimeClock>`,
+which uses datetime-like objects (specifically :class:`~pandas.Timestamp` and :class:`~pandas.Timedelta`) as the temporal units. The DateTimeClock
+can more easily facilitate the conversion rates to particular increments of time.
+
+Event Times
+-----------
+Discrete time simulations assume that all changes to a simulant's state vector happen at the 
+end of the time step, that is, the current clock time *plus* the step size. :mod:`vivarium` explicates this important distinction 
+and labels this quantity the *event time*. `Events <events_concept>` that correspond to (potential) state changes are mediated through the
+:class:`Event Manager <vivarium.framework.event.EventManager>`, which propagates events to :ref:`components <components_concept>` subscribed to them during particuar phases of the simulation lifecycle.
+The Event Manager uses the event time when calculating time-related outcomes, for example, age or year -dependent rates of morbidity and mortality.
+
+Time Interface
+--------------
+The Time plugin provides, via the :ref:`Builder <builder_concept>`, an :class:`interface <vivarium.framework.time.TimeInterface>` to access several clock methods that might be needed
+by other managers or components. In particular, components can access the current time and step size (and, implicitly, the event time).
+
+Individual Clocks
+-----------------
+:mod:`vivarium` also allows one to update simulants asynchronously with different frequencies depending on their state information.
+For example, a component that simulates the progression of a disease might need to update the state of each individual
+simulant at a different rate depending on the current state of the disease, for example, updating a simulant's state twice as frequently
+when infected than when in remission. The basic method is to give each simulant its own distinct clock time and step size instead of one global clock.
+
+Currently, the simulation still incorporates a global clock, which determines the start, end, and minimal step size of the simulation. The minimum step
+in term determines the maximum number of iterations that can occur in the simulation and serves to quantize the global step size. In each iteration of the simulation,
+the global clock is advanced to the earliest time in which a simulant must be updated, skipping several "minimum" timesteps if necessary. Simulants that are not scheduled to
+be updated in a particular iteration are simply excluded from the relevant events as propagated by the Event Manager.
+
+The Time Interface provides a method to modify a simulant's step size based on some criteria, :func:`builder.time.register_step_modifier() <vivarium.framework.time.TimeInterface.register_step_modifier>`.
```

Comment:
It occurs to me that perhaps this method ought to be called `register_step_size_modifier`.